commit dd3337ec10ff9b05fd22f862c2fff856d943da84
Author: Wenchao Xia <xiawenc@linux.vnet.ibm.com>
Date:   Fri Aug 3 15:36:59 2012 +0800

    fix id parsing with white space
    
      When device id string contains white space, parse is not correct.
    This patch fix it
    
    Signed-off-by: Wenchao Xia <xiawenc@linux.vnet.ibm.com>

diff --git a/libxkutil/device_parsing.c b/libxkutil/device_parsing.c
index 5393290..ceb4552 100644
--- a/libxkutil/device_parsing.c
+++ b/libxkutil/device_parsing.c
@@ -1033,7 +1033,7 @@ int parse_fq_devid(const char *devid, char **host, char **device)
 {
         int ret;
 
-        ret = sscanf(devid, "%a[^/]/%as", host, device);
+        ret = sscanf(devid, "%a[^/]/%a[^\n]", host, device);
         if (ret != 2) {
                 free(*host);
                 free(*device);
